app/controllers/assignments_controller.rb

```diff
@@ -50,8 +52,7 @@ def show
         end
         @grouping = @current_user.accepted_grouping_for(@assignment.id)
       end
-    end
-    unless @grouping.nil?
+    else
```

Comment:
I'm not sure why you changed this, my understanding is `@grouping` can be assigned on line 53.

Reply:
I'm not sure either